src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,206 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class HtmlTitleTests extends NewTestTemplate {
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @DataProvider
+  private Object[][] dataHtmlTitleTest001() {
+    return new Object[][]{
+        // original title
+        {
+            "sktest123",
```

Comment:
If you are thinking about having more context we can use more object-oriented approach and combine data in objects and use it (e.g. https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L457).